### ISSUE TITLE
Fine tune I3d

### ISSUE THREAD

**Issue by zpge** [Yuhan-Shen/ProTAS#6]
To improve the accuracy, I think it's better to combine the I3D model with your method and fine-tune them together in an end to end way. Do you think it's reasonable and possible?

**Comment by Yuhan-Shen** [comment]
I think it is possible and reasonable. We chose to use the pretrained I3D features to be consistent with prior works in temporal action segmentation. I believe fine-tuning the I3D model end to end may produce better performance as long as the computation/memory is not a bottleneck.
